### PR TITLE
Adjust food category bounds.

### PR DESCRIPTION
**Pull request by n1k0**
[User Story](https://www.notion.so/RESULTATS-Corriger-les-bornes-des-sous-scores-suite-investigation-sur-les-incoh-rences-e3ce58b448c148858710d09d38490516)

### CODE REVIEW

**Review by paulboosz — APPROVED** [review]
LGTM

**Comment by n1k0** [comment]
@magopian and @ccomb being off until Monday when I'll be off myself, I'm landing this.

File: tests/e2e-food.json [hunk 1]
```diff
@@ -266,38 +271,39 @@
       "wtu": 0.09704827341756544
     },
     "scoring": {
-      "category": "Toutes les catégories",
+      "category": "Fruits et légumes",
       "all": {
         "impact": 1230.7521714388874,
         "letter": "E",
-        "outOf100": 15
+        "outOf100": 0
```
Comment (ccomb) [inline tests/e2e-food.json:278]: Not sure about the computation. I should have missed something. The formula seems ok but If I apply for 1230 I get a score of 17, not 0

```
> ln = logBase e
> impact0=2270
> impact100=70.9
> score impact= 100 * (ln impact0 - ln impact) / (ln impact0 - ln impact100)
> score 1230
17.677982474506152 : Float
```

Comment (n1k0) [reply tests/e2e-food.json:278]: You missed the `"category=cakes"` param set, which will use the `cakes` category bounds, which are `{ impact100 = 100, impact0 = 700 }`.

**Review by ccomb — COMMENTED** [review]
I understand the mango by plane having a score of 0 :-)

**Pull request merged**